docs/content/api/java-api.md

```diff
@@ -600,4 +619,21 @@ public class StreamWriteTable {
 | map          | org.apache.paimon.data.InternalMap   |
 | InternalRow  | org.apache.paimon.data.InternalRow   |
 
+## Predicate Types
+
+| SQL Predicate| Paimon Predicate                           | 
+|:-------------|:-------------------------------------------|
+| and          | org.apache.paimon.predicate.And            |
```

Comment:
Maybe we can just `PredicateBuilder.and`?

Reply:
I have updated this PR. @JingsongLi PTAL.